README.md

```diff
@@ -1,2 +1,429 @@
+      --source-custom-catalog-impl=<customCatalogImpl>
+                     Optional fully qualified class name of the custom catalog implementation of the source catalog. Required when the catalog type
+                       is CUSTOM.
+                     Example: --source-custom-catalog-impl org.apache.iceberg.AwesomeCatalog
+Target catalog options:
+      --target-catalog-type=<type>
+                     Target catalog type. Can be one of these [CUSTOM, DYNAMODB, ECS, GLUE, HADOOP, HIVE, JDBC, NESSIE, REST].
+                     Example: --target-catalog-type GLUE
+                              --target-catalog-type NESSIE
+      --target-catalog-properties=<String=String>[,<String=String>...]
+                     Iceberg catalog properties for target catalog (like uri, warehouse, etc).
+                     Example: --target-catalog-properties uri=http://localhost:19120/api/v1,ref=main,warehouse=/tmp/warehouseNessie
+      --target-catalog-hadoop-conf=<String=String>[,<String=String>...]
+                     Optional target catalog Hadoop configurations (like fs.s3a.secret.key, fs.s3a.access.key) required when using an Iceberg FileIO.
+                     Example: --target-catalog-hadoop-conf fs.s3a.secret.key=$SECRETKEY,fs.s3a.access.key=$ACCESSKEY
+      --target-custom-catalog-impl=<customCatalogImpl>
+                     Optional fully qualified class name of the custom catalog implementation of the target catalog. Required when the catalog type
+                       is CUSTOM.
+                     Example: --target-custom-catalog-impl org.apache.iceberg.AwesomeCatalog
+Identifier options:
+      --identifiers=<identifiers>[,<identifiers>...]
+                     Optional selective set of identifiers to register. If not specified, all the tables will be registered. Use this when there are
+                       few identifiers that need to be registered. For a large number of identifiers, use the `--identifiers-from-file` or
+                       `--identifiers-regex` option.
+                     Example: --identifiers foo.t1,bar.t2
+      --identifiers-from-file=<identifiersFromFile>
+                     Optional text file path that contains a set of table identifiers (one per line) to register. Should not be used with
+                       `--identifiers` or `--identifiers-regex` option.
+                     Example: --identifiers-from-file /tmp/files/ids.txt
+      --identifiers-regex=<identifiersRegEx>
+                     Optional regular expression pattern used to register only the tables whose identifiers match this pattern. Should not be used
+                       with `--identifiers` or '--identifiers-from-file' option.
+                     Example: --identifiers-regex ^foo\..*
+```
+
+Note: Options for migrate command is exactly same as register command.
+
+> :warning: **It is recommended to use this CLI tool when there is no in-progress commits for the tables in the source catalog.**
```

Comment:
Really? Nothing changed here.

Reply:
previously this statement was not there. I added it in the end.

`Which can lead to missing updates, loss of data and table corruption.`

Reply:
I will update this with the new strong sentence from chatGPT

`Avoid using this CLI tool when there are in-progress commits for tables in the source catalog to prevent missing updates, data loss, and table corruption in the target catalog. In-progress commits may not be properly transferred and could compromise the integrity of your data.`

Reply:
I think you checked only the latest commits. These were addressed before that commit 

678d5ca057e3aa5bde02fe6a618148af5f3c3cf6 is the commit which fixed it.